src/components/CategoryFilter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const OPTIONS = [
+  { name: 'All', id: 1 },
+  { name: 'Action', id: 2 },
+  { name: 'Biography', id: 3 },
+  { name: 'History', id: 4 },
+  { name: 'Horror', id: 5 },
+  { name: 'Kids', id: 6 },
+  { name: 'Learning', id: 7 },
+  { name: 'Sci-Fi', id: 8 },
+];
+
+const CategoryFilter = props => {
+  const { handleChange } = props;
+  const optionElements = OPTIONS.map(({ name, id }) => (
+    <option key={id}>{name}</option>
+  ));
+  return <select onChange={handleChange.bind(this)}>{optionElements}</select>;
```

Comment:
- [ ] **[OPTIONAL]** There is no need to bind functions in functional components. Actually there is no `this` inside JS functions. Kindly check out [this](https://stackoverflow.com/questions/53215067/how-can-i-bind-function-with-hooks-in-react/53246229#53246229) for more info.